SU2_PY/SU2/eval/functions.py

```diff
@@ -222,7 +222,18 @@ def aerodynamics( config, state=None ):
     name = files['MESH']
     name = su2io.expand_part(name,config)
     link.extend(name)
-    
+
+    # files: restarts
+    if config.get('TIME_DOMAIN', 'NO') == 'YES' and config.get('RESTART_SOL','NO') =='YES':
+        if  'RESTART_FILE_1' in files: # not the case for directdiff restart
+            name = files['RESTART_FILE_1']
+            name = su2io.expand_part(name, config)
+            link.extend(name)
+        if 'RESTART_FILE_2' in files:
```

Comment:
Will this work for 1st order time integration?

Reply:
Not right now. The main script `shape_optimization.py` exits if there are less than two restart options. The code snipped above would work though. 
Which config option is used to specify the order of time integration? Then, I can add this feature.